MergerService/Runners/IMainRunner.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace MergerService.Runners
+{
+    public interface IMainRunner
```

Comment:
instead of beeing defined in a separate file you can put it above the class definition in the MainRunner file

Reply:
IMO this is fine just as we did for different interfaces in this project (such as ITileMerger).

Reply:
I'm leaving it as is since it is defined this way for other existing interfaces, if you decide on a different approach, let me know